tamr_unify_client/models/project/estimated_pair_counts.py

```diff
@@ -20,15 +20,15 @@ def is_up_to_date(self) -> bool:
     def total_estimate(self) -> dict:
         """The total number of estimated candidate pairs and generated pairs for the model across all clauses.
```

Comment:
You state in the `:type:` that this is a `dict[str, int]`, but could you add a description of what those `str` and `int` are? E.g. for a made up `dict[int, int]` I could say "A mapping from integers in the range 1-10 to their squares. x -> x^2". Could you do the analogous thing for this function?